cpp/include/cudf/aggregation.hpp

```diff
@@ -305,34 +306,34 @@ std::unique_ptr<Base> make_row_number_aggregation();
  *  3. `RANK` aggregations are not compatible with exclusive scans.
  *
  * @code{.pseudo}
- * Example: Consider an motor-racing statistics dataset, containing the following columns:
- *   1. driver_name:   (STRING) Name of the car driver
- *   2. num_overtakes: (INT32)  Number of times the driver overtook another car in a lap
- *   3. lap_number:    (INT32)  The number of the lap
+ * Example: Consider a motor-racing statistics dataset, containing the following columns:
+ *   1. venue:  (STRING) Location of the race event
+ *   2. driver: (STRING) Name of the car driver (abbreviated to 3 characters)
+ *   3. time:   (INT32)  Time taken to complete the circuit
  *
  * For the following presorted data:
  *
- *  [ // driver_name,  num_overtakes,  lap_number
- *    {   "bottas",        2,            3        },
- *    {   "bottas",        2,            7        },
- *    {   "bottas",        2,            7        },
- *    {   "bottas",        1,            1        },
- *    {   "bottas",        1,            2        },
- *    {   "hamilton",      4,            1        },
- *    {   "hamilton",      4,            1        },
- *    {   "hamilton",      3,            4        },
- *    {   "hamilton",      2,            4        }
+ *  [ //      venue,           driver,           time
+ *    {   "silverstone",  "HAM" ("hamilton"),   15823},
+ *    {   "silverstone",  "LEC" ("leclerc"),    15827},
+ *    {   "silverstone",  "BOT" ("bottas"),     15834},  // <-- TIE!
```

Comment:
What is "TIE"?

Reply:
In this example, at the race in `silverstone`, `bottas` and `norris` were tied for third place, for finishing the race in the exact same time.
Note that this is an example for illustration only. FWIW, at the [actual race in 2021](https://en.wikipedia.org/wiki/2021_British_Grand_Prix#Race_classification), Lando Norris was 17 seconds behind Bottas.

Reply:
I can replace this with:
```suggestion
 *    {   "silverstone",  "BOT" ("bottas"),     15834},  // <-- Tied for 3rd place.
```

Reply:
The longer comment is slightly more helpful.

Reply:
Done. I hadn't realized that the phrasing would be confusing. Fixed now.